static/js/biomedical/protein/types.ts

```diff
@@ -32,8 +32,8 @@ export interface ProteinNode extends SimulationNodeDatum {
 
 // https://github.com/tomwanzek/d3-v4-definitelytyped/blob/06ceb1a93584083475ecb4fc8b3144f34bac6d76/src/d3-force/index.d.ts#L24
 export interface InteractionLink extends SimulationLinkDatum<ProteinNode> {
-  sourceId: string;
-  targetId: string;
+  source: string;
```

Comment:
I had messed this up originally.  (Code was still working before without type errors because SimulationLinkDatum also has source, target)